MAINTAINERS.yml

```
@@ -0,0 +1,1050 @@
+        - dts/bindings/i2s/
+        - include/i2s.h
+        - include/drivers/i2s.h
+        - tests/drivers/i2s/
+    labels:
+        - "area: I2S"
+
+"Drivers: LED":
+    # Provisional entry, to be corrected
+    status: orphaned
+    collaborators:
+        - Mani-Sadhasivam
+    files:
+        - drivers/led/
+        - include/led.h
+        - include/drivers/led/
+        - include/drivers/led.h
+        - samples/drivers/led_*/
+    labels:
+        - "area: LED"
+
+"Drivers: LED Strip":
+    status: maintained
+    maintainers:
+        - mbolivar
+    files:
+        - drivers/led_strip/
+        - dts/bindings/led_strip/
+        - include/led_strip.h
+        - include/drivers/led_strip.h
+
+"Drivers: Modem":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - mike-scott
+    files:
+        - drivers/modem/
+    labels:
+        - "area: Modem"
+
+"Drivers: Neural networking":
+    # Provisional entry, to be corrected
+    status: orphaned
+    files:
+        - drivers/neural_net/
+
+"Drivers: PCI":
+    # Provisional entry, to be corrected
+    # TODO: spreadsheet not matching CODEOWNERS
+    status: maintained
+    maintainers:
+        - dcpleung
+    collaborators:
+        - andrewboie
+    files:
+        - drivers/pcie/
+        - include/drivers/pcie/
+    labels:
+        - "area: PCI"
+
+"Drivers: Pinmux":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - mnkp
+    files:
+        - doc/reference/peripherals/pinmux.rst
+        - drivers/pinmux/
+        - include/drivers/pinmux.h
+        - include/pinmux.h
+        - tests/drivers/pinmux/
+    labels:
+        - "area: Pinmux"
+
+"Drivers: PWM":
+    # Provisional entry, to be corrected
+    status: orphaned
+    files:
+        - drivers/pwm/
+        - dts/bindings/pwm/
+        - include/drivers/pwm.h
+        - include/pwm.h
+        - tests/drivers/pwm/
+    labels:
+        - "area: PWM"
+
+"Drivers: Serial/UART":
+    status: maintained
+    maintainers:
+        - dcpleung
+    files:
+        - drivers/serial/
+        - dts/bindings/serial/
+    labels:
+        - "area: UART"
+
+"Drivers: Sensors":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - MaureenHelm
+    files:
+        - drivers/sensor/
+        - include/drivers/sensor.h
+        - samples/sensor/
+    labels:
+        - "area: Sensors"
+
+"Drivers: SPI":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - tbursztyka
+    files:
+        - drivers/spi/
+        - include/drivers/spi.h
+        - tests/drivers/spi/
+    labels:
+        - "area: SPI"
+
+"Drivers: System timer":
+    # Provisional entry, to be corrected
+    # TODO: maintainers spreadsheet not matching CODEOWNERS
+    status: maintained
+    maintainers:
+        - andyross
+    files:
+        - drivers/timer/
+    labels:
+        - "area: Timer"
+
```

```suggestion
"Drivers: USB STM32":
    # Provisional entry, to be corrected
    collaborators:
        - ydamigos
        - loicpoulain
    files:
        - drivers/usb/device/usb_dc_stm32.c
        - dts/bindings/usb/st,stm32-*
```
@ydamigos, @loicpoulain, can you confirm ?

---

Yes, Ok for me.

---

Disagree. Suggest:

```diff
+"Drivers: USB":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - jfischer-phytec-iot
+    collaborators:
+        - finikorg
+    files:
+        - drivers/usb/device/*
+        - dts/bindings/usb/*
```

---

@jfischer-phytec-iot, can you elaborate?
Is there an incompatibility in having maintainer on a driver (as a subsystem) and having collaborators on a particular driver (of this subsystem)?